bio/macs2/callpeak/environment.yaml

```
@@ -0,0 +1,5 @@
+channels:
+  - bioconda
+  - defaults
```

Macs2 depends on a number of `conda-forge` packagess, so I think this should be part of the mix, here:
```suggestion
  - conda-forge
  - bioconda
  - defaults
```